templates/stub.erb

```diff
@@ -3,3 +3,6 @@ stub-zone:
 <% @address.each do |addr| -%>
   stub-addr: <%= addr %>
 <% end -%>
+<% if @no_cache -%>
```

Comment:
if no_cache == 'false' this will pass as a none empty string is considered true

```
$ ruby -e "if 'false' then puts 'true' end "                                                               
true
```

Reply:
ya, but we won't get here with an empty string, given our type definition

Reply:
oh, right, we don't normalise `'false'` to `false`

Reply:
Ah yes, clearly I've not been writing much ruby.  I'll fix.